frontend/src/metabase/timelines/common/components/EventForm/EventForm.tsx

```diff
@@ -29,22 +33,31 @@ const EventSchema = Yup.object({
   timeline_id: Yup.number(),
 });
 
-export interface EventFormProps {
+export interface EventFormOwnProps {
   initialValues: TimelineEventData;
   timelines?: Timeline[];
+  formattingSettings?: FormattingSettings;
   onSubmit: (data: TimelineEventData) => void;
   onArchive?: () => void;
   onCancel?: () => void;
 }
 
+export interface EventFormStateProps {
+  formattingSettings?: FormattingSettings;
+}
+
+export type EventFormProps = EventFormOwnProps & EventFormStateProps;
+
```

Comment:
Seems like `formattingSettings` are defined in both `OwnProps` and `StateProps` now. Can we keep only one?

Reply:
Fixed